dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java

```diff
@@ -877,20 +845,52 @@
     public static final String METRICS_KEY = "metrics";
     public static final String METRICS_PORT = "metrics.port";
     public static final String METRICS_PROTOCOL = "metrics.protocol";
+    // END dubbo-monitor-api
+
+    // BEGIN dubbo-metadata-report-api
+    public static final String METADATA_REPORT_KEY = "metadata";
+
+    public static final String RETRY_TIMES_KEY = "retry.times";
 
+    public static final Integer DEFAULT_METADATA_REPORT_RETRY_TIMES = 100;
+
+    public static final String RETRY_PERIOD_KEY = "retry.period";
+
+    public static final Integer DEFAULT_METADATA_REPORT_RETRY_PERIOD = 3000;
+
+    public static final String SYNC_REPORT_KEY = "sync.report";
+
+    public static final String CYCLE_REPORT_KEY = "cycle.report";
+
+    public static final Boolean DEFAULT_METADATA_REPORT_CYCLE_REPORT = true;
+    // END dubbo-metadata-report-api
+
+    // BEGIN dubbo-filter-cache
+    public static final String CACHE_KEY = "cache";
+    // END dubbo-filter-cache
+
+
+    // BEGIN dubbo-filter-validation
+    public static final String VALIDATION_KEY = "validation";
+    // END dubbo-filter-validation
+
+    public static final String DEFAULT_CLUSTER = "failover";
```

Comment:
This is not used, it is recommended to delete, or put it in dubbo-cluster

Reply:
let's do it later. will not do it in step1.